### PR TITLE
Update mender binary delta v1.0.0 thud

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta_1.0.0.bb [hunk 1]
```diff
@@ -1,12 +1,13 @@
 LICENSE = "CLOSED"
 LICENSE_FLAGS = "commercial"
 
-SUB_FOLDER_arm = "armhf"
+SUB_FOLDER_arm = "arm"
```
Comment (oleorhagen) [inline meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta_1.0.0.bb]: Curious: Why was it named with the hard-float extension in the first place?

Comment (lluiscampos) [reply meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta_1.0.0.bb]: We were following Debian naming, but we decided to switch to Yocto.

ref https://github.com/mendersoftware/mender-binary-delta/commit/e9fa48f5cb99bc1a91dcb2d75f6cce77a24f74fc
ref https://www.debian.org/ports/

Comment (oleorhagen) [reply meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta_1.0.0.bb]: Alright. Thanks

**Comment by lluiscampos** [comment]
This PR was delayed due to problems with the CI, basically requiring https://github.com/mendersoftware/mender-qa/pull/330.

Successfully built now. Merging!

**Pull request merged**